src/v2/cookbook/packaging-sfc-for-npm.md

```diff
@@ -43,9 +43,9 @@ Es verdad, usted puede compartir archivos `.vue` directamente y cualquiera usand
 
 Empaquetando apropiadamente tu distribución de SFC vía npm  permite que a tu compoenente ser compartido de una manera en la cuál está listo para usarlo donde sea!
```

Comment:
I Know this line was not modified in this PR, but since it is already translated I'd take the opportunity to also fix it and refactor the way it was written:

Original text: 
```
Properly packaging your SFC for distribution via npm enables your component to be shared in a way which is ready to use everywhere!
```

```suggestion
El empaquetado adecuado de su SFC para su distribución a través de npm permite que su componente se comparta de una manera que está lista para usar en cualquier lugar.
```

What do you think?

Reply:
I think is ok!!!